shapely/geoarrow.py

```diff
@@ -0,0 +1,163 @@
+import enum
+
+import numpy as np
+
+from shapely import GeometryType
+from shapely._geoarrow import (
+    SchemaCalculator,
+    ArrayBuilder,
+    ArrayReader,
+    GeoArrowGEOSException,
+)  # NOQA
+
+
+class Encoding(enum.Enum):
+    WKB = SchemaCalculator.ENCODING_WKB
+    WKT = SchemaCalculator.ENCODING_WKT
+    GEOARROW = SchemaCalculator.ENCODING_GEOARROW
```

Comment:
Does this default to separated encoding? Since GEOS is interleaved I have been defaulting to interleaved encoding for all my Python APIs (And I need interleaved for lonboard). But not sure which we should suggest.